webrtc/videoroom/assets/src/membraneWebRTC.ts

```diff
@@ -226,16 +226,15 @@ export class MembraneWebRTC {
   private onTrack = () => {
     return (event: RTCTrackEvent) => {
       const [stream] = event.streams;
-      const mid = event.transceiver.mid!;
-      const isScreenSharing = mid.includes("SCREEN") || false;
+      const id = event.track.id;
+      const isScreenSharing = id.includes("SCREEN") || false;
```

Comment:
Have you checked that screensharing works? I tried to wrap my head around this pr and one in webrtc plugin and got lost easily...